.github/workflows/continuous-monitoring.yml

```diff
@@ -11,9 +11,22 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
+      
       - uses: actions/setup-go@v2
         with:
           go-version: '^1.16'
+      
       - name: Run distribution test
+        id: distribution-availability
         run: go test distributioncheck
         working-directory: integration-tests/distributioncheck
+      
+      - name: Publish metric on X-Ray Go SDK distribution availability
+        if: ${{ always() }}
+        run: |
+          if [[ "${{ steps.distribution-availability.outcome }}" == "failure" ]]; then
+            aws cloudwatch put-metric-data --metric-name XRayGoSDKDistributionAvailability --dimensions failure=rate --namespace MonitorSDK --value 1 --timestamp $(date +%s)
```

Comment:
How do we know the AWS CLI is installed?

Reply:
Github instance installs the latest version of AWS CLI by default.

Reply:
Don't we need to add aws login? Also what account are we going to use? Ideally we use https://github.com/aws-observability/aws-otel-ops to set up the secrets.

Reply:
we need to configure aws credentials on Github instance. We are using the testing account for this so we can use secrets that are set up for integration test workflow no need to set up another secrets.